.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-*    @susanharper-okta @kimmaida-okta @janethu-okta @grahamsmith-okta @barbaravo-okta @MarkCowan-okta @emikhasyak-okta @paulwallace-okta
+*    @susanharper-okta @kimmaida-okta @janethu-okta @grahamsmith-okta @barbaravo-okta @emikhasyak-okta @paulwallace-okta
+/acrolinx/global-config.edn @janethu-okta @susanharper-okta
```

Comment:
`/.github/acrolinx/global-config.edn` would be a complete path? `/` is the root of the repo, everything after that is from root.